src/index.html

```diff
@@ -17,11 +17,399 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
Link to a SCSS file is incorrect. In HTML, you should link to a CSS file. Please compile your SCSS to CSS and link to the resulting CSS file.